common/hooks/useFocusTrap.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import { useEffect, type ElementRef } from 'react';
+
+type TrapElRef = {|
+  current: ElementRef<any>,
+|};
+
+export default (startRef: ?TrapElRef, endRef: ?TrapElRef) => {
```

Comment:
Should we follow the components pattern, which I think helps with debugging as the function name can be spat out when sourcemaps work of

```JS
function useFocusTrap() {}
export default useFocusTrap;
```

Reply:
yeah, good shout. Will update.